lms/envs/devstack.py

```diff
@@ -228,3 +228,15 @@ def should_show_debug_toolbar(_):
 MODULESTORE = convert_module_store_setting_if_needed(MODULESTORE)
 
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+
+#####################################################################
+# Mobile
+FEATURES['ENABLE_COURSE_BLOCKS_NAVIGATION_API'] = True
+FEATURES['ENABLE_RENDER_XBLOCK_API'] = True
```

Comment:
Ah. I tried to run it without those and was getting URL resolver errors. Which flags are necessary?